packages/plugin-ext/src/main/browser/notification-main.ts

```diff
@@ -54,7 +54,8 @@ export class NotificationMainImpl implements NotificationMain, Disposable {
     protected mapOptions(options: string | NotificationMain.StartProgressOptions): ProgressMessage {
         const text = typeof options === 'string' ? options : options.title;
         const location = typeof options === 'string' ? 'notification' : options.location;
-        return { text, options: { location, cancelable: true } };
+        const cancelable = typeof options === 'string' ? true : (options.cancellable === undefined || options.cancellable);
```

Comment:
When do we pass `options` as a string and why when it should cancellable? I could not find any clients.

Reply:
good question... lemme check

Reply:
https://github.com/eclipse-theia/theia/commit/240f67e5216ed30bd99fbc8ea2e7d04c9db3305a#diff-7eea09f2b0c581de589cb87e1127fb80R666

I hope the link works.
I bet, I did want to avoid a braking change :-(

Reply:
Should I clean up and remove from there? Would that then require to make a change log?

Reply:
I don't think this code is used anywhere. It would be fine to clean up, otherwise it is confusing.

Reply:
this shouldn't break anything. just updated it. waiting to next build.